cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -1116,4 +1104,18 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         declStmt.addDeclaration(decl)
         return declStmt
     }
+
+    /**
+     * Checks whether [mightBeAsync] has the [IsAsync] property and adds a warning to the
+     * corresponding [parentNode] stored in [Node.additionalProblems].
+     */
+    private fun addAsyncWarning(mightBeAsync: Python.AST.AsyncOrNot, parentNode: Node) {
```

Comment:
I'd call it simply `node` for simplicity

Reply:
I wanted to make it perfectly clear that this is not the node we are testing `async` on. I'll change this, if you insist, however, I prefer it this way. Or maybe `reportingNode`??

Reply:
Since there's only one node involved, it should be pretty clear already but I'm also ok like this